spec/text-editor-presenter-spec.coffee

```diff
@@ -611,7 +611,7 @@ describe "TextEditorPresenter", ->
             expect(presenter.getState().hiddenInput.width).toBe 15
 
             expectStateUpdate presenter, ->
-              presenter.getLinesYardstick().setScopedCharacterWidth(['source.js', 'storage.modifier.js'], 'r', 20)
+              presenter.getLinesYardstick().setScopedCharacterWidth(['source.js', 'storage.type.var.js'], 'r', 20)
```

Comment:
Basically to test the integration with `LinesYardstick` and make sure `TextEditorPresenter` delegates characters coordinates computation to that object instead of doing it on its own (i.e. as we did before measuring from the DOM).

This is dependent on how we tokenize stuff, hence the required change: I wonder if all these tests (and the other ones that use `language-javascript`) should depend on a different, fake language package so that we don't have to perform this annoying task of keeping them up-to-date.